pom.xml

```diff
@@ -69,7 +69,7 @@
     <tony.version>0.3.25</tony.version>
     <jersey.version>2.27</jersey.version>
     <jersey.servlet.version>1.13</jersey.servlet.version>
-    <jetty.version>9.4.18.v20190429</jetty.version>
+    <jetty.version>9.4.35.v20201120</jetty.version>
```

Comment:
Let's merge it. But there's new Jetty CVE affecting versions below 9.4.39. We can raise another jira to update again.